docs/en/observability/monitor-k8s/monitor-k8s-overview.asciidoc

```diff
@@ -67,11 +37,16 @@ with cloud, Kubernetes, and host metadata.
 
 image::images/metadata-processors.png[Metadata processors for cloud, Kubernetes, and host metadata]
 
+For more on these processors, refer to the {fleet-guide}/add-cloud-metadata-processor.html[`add_cloud_metadata`], {fleet-guide}/add_kubernetes_metadata-processor.html[`add_kubernetes_metadata`], and {fleet-guide}/add_host_metadata-processor.html[`add_host_metadata`] documentation.
+
```

Comment:
The kubernetes integration enriches by default logs and metrics with valuable metadata for the related resources. 

For metrics:
- All Kubernetes metrics are enriched by default with metadata. The enrichment is based purely in code and can be configured with `add_resource_metadata` block per dataset

Eg. In Fleet:
Open Kube-State-metrics> State_Pod Dataset > `Add node and namespace metadata`
```
add_resource_metadata:
  namespace:
   enabled: true
    #use_regex_include: false
    include_labels: ["namespacelabel1"]
    #use_regex_exclude: false
    #exclude_labels: ["namespacelabel2"]
  node:
   enabled: true
    #use_regex_include: false
    include_labels: ["nodelabel2"]
    include_annotations: ["nodeannotation1"]
    #use_regex_exclude: false
    #exclude_labels: ["nodelabel3"]
  #deployment: false
  #cronjob: false
```

- `add_kubernetes_metadata` processor is advised not to be used with latest versions of Kubernetes integration as enrichment is already happening in background
- `add_metadata` flag can be use to toggle on/off metada enrichment only for metrics


For Logs:
- All Kubernetes logs are enriched by default with metadata. The enrichment is based on the kubernetes provider that is enabled by default. It can be configured with `add_resource_metadata` block of the https://www.elastic.co/guide/en/fleet/current/kubernetes-provider.html[provider] 
- `add_kubernetes_metadata` processor is advised not to be used for lohs enrichment with latest versions of Kubernetes integration as enrichment is already happening in background

Reply:
@gizas if add_kubernetes_metadata isn't recommended at this point, should we just remove it from this section?

Reply:
I thought it might be better to put some of the information about configuring metadata enrichment in the sections about [kubelet API metrics](https://github.com/elastic/observability-docs/pull/3581/files#diff-bda111cbc6c4e1d74a590c82bcd89c2c72b7f983a4761efa6cc1f8d8a41a8146R105-R122) and [kube-state metrics](https://github.com/elastic/observability-docs/pull/3581/files#diff-bda111cbc6c4e1d74a590c82bcd89c2c72b7f983a4761efa6cc1f8d8a41a8146R174-R201) if you wouldn't mind taking a look and making sure that makes sense.

Reply:
Totally makes sense. Those are the ones affected from enrichemnt by configuring add_resource_metadata block

Just we need to include the reference for the logs collection that is only affected from provider. Under container_logs maybe?

> if add_kubernetes_metadata isn't recommended at this point, should we just remove it from this section?
This is for the processor. Yes I agree

Indeed we can remove. If you want you can add a link in the end, with something like a note: `In cases where kubernetes Integration is not enabled and you still want kubernetes metadata enrichment in your integration, the relevant  add_kubernetes_metadata  processor can be used` and add a link